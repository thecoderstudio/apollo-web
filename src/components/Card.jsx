import styled from 'styled-components';

const Card = styled.div`
  box-shadow: ${(props) => props.theme.boxShadow};
  padding: 40px;
  border-radius: 10px;
  background-color: ${(props) => props.theme.lightBlack};
`;

export default Card;
