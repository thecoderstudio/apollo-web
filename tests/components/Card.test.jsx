import React from 'react';
import renderer from 'react-test-renderer';
import Card from '../../src/components/Card';

it('renders correctly', () => {
  const tree = renderer
    .create(<Card>Test</Card>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
