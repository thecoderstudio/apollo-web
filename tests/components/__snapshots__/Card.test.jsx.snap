// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c0 {
  box-shadow: 0 10px 15px -3px rgba(0,0,0,0.1),0 4px 6px -2px rgba(0,0,0,0.05);
  padding: 40px;
  border-radius: 10px;
}

<div
  className="c0"
>
  Test
</div>
`;
